packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -292,6 +309,32 @@ async function transformGraphQLSchema(context, options) {
     transformerList.push(new SearchableModelTransformer());
   }
 
+  const transformerConfigPath = path.join(resourceDir, TRANSFORM_CONFIG_FILE_NAME);
+  const customTransformersConfig = loadCustomTransformersConfig(transformerConfigPath, context);
+  const customTransformers =
+    (customTransformersConfig && customTransformersConfig.transformers
+      ? customTransformersConfig.transformers
+      : [])
+      .map((transformer) => {
+        const fileUrlMatch = /^file:\/\/(.*)\s*$/m.exec(transformer);
+        const modulePath = fileUrlMatch ? fileUrlMatch[1] : transformer;
+        // handle 'cannot find module'
+        try {
+          const imported = require(modulePath);
+          const CustomTransformer = imported.default;
+          return CustomTransformer.call({});
```

Comment:
What do you think about separating the loading and execution of the custom transformer? The ```catch``` handles the loading issues, but if an execution error happens, then the error message will be misleading.